### DISCUSSION TITLE
Moving to nightly channel of clippy?

### DISCUSSION THREAD

**Discussion by drHuangMHT** [libp2p/rust-libp2p#5280] [General]
Clippy beta is quite buggy recently so would it be better if we change the test to use nightly channel?

**Comment by jxs** [comment]
Hi! Have you read https://github.com/libp2p/rust-libp2p/issues/2951#issue-1390138614? I'd prefer to keep `beta` as it's a not required test, so even if it's buggy and flaky it fulfills its purpose which is to anticipate with a window of 6 months lints that will break the CI, how would we achieve the same with nightly?
